audit/get-sandbox-user-emails.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Get a list of all orgs names prepended with the word sandbox"
+orgs=($(cf orgs | grep '^sandbox'))
+
+echo "Username, Org GUID"
+
+# Loop through the array and look up quota details.
+for org in "${orgs[@]}"
+do
+    guid=$(cf org $org --guid)
+
+    emails=($(cf curl "/v2/organizations/$guid/users" | jq '.resources | .[].entity | .username'))
```

Comment:
We might want to shift this to the `v3` API call - that may change the response output though, so double check that it's what you expect!

Reply:
That's fair. I'll merge this, but will create an issue to update the script for the newer version of the CF API.

Reply:
Sounds good, thanks @mheadd!